### ISSUE TITLE
Hacer MENÚ!!!

### ISSUE THREAD

**Issue by fernandouval** [datauy/ElijoEstudiar#29]

**Comment by danielcarranza** [comment]
@despinoUY Che, me parece que el ícono de menú y la cruz tienen mal el CSS, demasiado grueso el trazo, el color y sin border radius, ¿Puede ser?

**Comment by despinoUY** [comment]
Sí, tenés razón, voy a revisar cuanto tiene de ancho en los diseños de Agustín. Lo mismo del border radius.

**Comment by danielcarranza** [comment]
En Zeplin sacás el CSS tal cual

**Comment by danielcarranza** [comment]
No te olvides de ésto cuando puedas @despinoUY
